src/openrct2-ui/windows/Park.cpp

```diff
@@ -681,13 +681,18 @@ static constexpr WindowParkAward _parkAwards[] = {
 #pragma region Rating page
         void OnResizeRating()
         {
-            WindowSetResize(*this, 255, 182, 255, 182);
+            flags |= WF_RESIZABLE;
```

Comment:
This flag isn't removed when switching back to the other tabs, is it?

Reply:
It's not, however the downstream logic compares this flag and the min and max sizes to decide if the window is resizeable. The non-resizeable tabs have the min and max set to the same value and therefore there is no resizability. 

So maybe there's an argument that the resize flag should be removed as being superfluous. But on the other hand, perhaps there are some windows that might want to temporarily disable resizability without changing the minimum and maximum size values.